rules/storyboarding-do-you-conduct-specification-analysis-by-creating-mock-ups/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
-title: Storyboarding - Do you conduct specification analysis by creating mock-ups?
-uri: storyboarding-do-you-conduct-specification-analysis-by-creating-mock-ups
+title: Do you conduct specification analysis by creating mock-ups?
```

Comment:
@Geordie88 

See the email thread **Re: Test please - storyboarding rule** 

I requested a title change + URL change